code/modules/hippie/sound/pooling.dm

```diff
@@ -0,0 +1,145 @@
+#define POOL_HIT_COUNT 1
+#define POOL_MISS_COUNT 2
+#define POOLINGS 3
+#define UNPOOLINGS 4
+#define EVICTIONS 5
+#define DEFAULT_POOL_SIZE 50
+
+#ifdef DETAILED_POOL_STATS
+var/global/list/pool_stats = list()
+
+proc/increment_pool_stats(var/type, var/index)
+	if(!type || index < 1 || index > 5)
+		return
+	var/list/L = pool_stats[type]
+	if(!L)
+		L = list(0,0,0,0,0)
+
+		pool_stats[type] = L
+
+	L[index]++
+
+
+#endif
+
+
+/datum/var/tmp/disposed = 0
+/datum/var/tmp/qdeled = 0
+
+/datum/Del()
```

Comment:
This is also bad, we don't need this to go into our pool, /tg/ removed pooling for a reason, and I'm quite on the fence if we actually need pooling for sounds to be honest.

You might want to try this without pooling